src/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 900;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  padding: 0;
+  font-style: normal;
+  outline: none;
+}
+
+.search-input {
+  padding-left: 62px;
+  width: 100%;
+  height: 70px;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
```

Comment:
Check this [link](https://9elements.com/css-rule-order/) to organize your CSS properties in the right order